src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+
+## Suit charge
+
+Currently, the only real downside for wearing a hardsuit 24/7 is a movement speed penalty that needs to stay around 10-30%, or else you'll end up being too slow to do anything and the game will be unfun. This has led to people wearing hardsuits constantly in anticipation of something happening, which isn't interesting at all. If everyone has space protection, what's the point in space?
+
+A powered hardsuit would still slow you down somewhat, you are wearing a bulky metal suit after all. But movement speed is no longer the only downside to wearing hardsuits. As a result, contributors will have more freedom to make certain suits mobile while still having power charge as a balancing factor to make up for it.
+
+Powered hardsuits would start the round charged to 100% (how much energy a suit can hold varies), but do not use power cells. Instead, they would have to be charged at a suit storage. You should be forced to remove your hardsuit to charge it, you cannot just jump into a suit storage to use it. Every so often while using your suit, you'll have to stop back at your department to charge them or ask a nearby department to open up and let you charge with their storages.
+
+This exists to keep people from wearing them 24/7 as people did with modsuits, using less abilities would drain charge slower, allowing people to disable things when unneeded to conserve power, or just never use them for extended spacesuit time. The same applies to antagonists, though your blood-red hardsuit has a much higher energy capacity than any of the crew's hardsuits, you'll still have to charge it at some point, and when that time comes you'll be extremely vulnerable. No more syndies buying a jetpack and a blood-red and just flying around the station for the entire round shooting people.
+
+Running out of charge would not prevent the suit from protecting you from space, but it would give you a major movement speed decrease (think 25%-75% depending on the suit) and entirely prevent you from using ANY abilities. Stored items will be locked inside, magboots will be turned off, and additional armor shields will be disabled. Engineers could still wrap up repairs after running out of power if they have other equipment rather than relying on their suits, and security could finish dragging an apprehended suspect back to their department, just very slowly.
+
+## Suit Abilities
+
+Specific hardsuits would have specific abilities that could be seen in SS13 modsuits, but these abilities would be hard-locked to specific suits, rather than being placed on any suit with a module. The abilities could include...
+
+- **A basic flashlight on the helmet/suit.**
+A small light with a low energy cost that lets you see. You could keep it off in a lit area, or if you want to conserve suit charge. Carrying a seclight or a normal flashlight is also viable.
+- **Engi hardsuits giving welding protection.**
```

Comment:
Engi hardsuits having radiation dampeners that eat up a shitton of power when active.

Let's regular engineers refill Singulo without giving them cancer instantly, but ensures they can't just chill out there to avoid everyone, they would have to put the suit back. You could scale power usage based on the amount of rads received. 

CE suit can either just get inherent rad immunity like it always did (because it's a one time suit) or a significantly reduced power consumption, something inconsequential, CE goes through a lot.